### PR TITLE
Restored bluetooth prompt (fixes #174)

### LINKED ISSUE #174: [Notifications] Android support
Following discussions on #14 we will need to implement android notifications.

### PR DESCRIPTION
**Pull request by wilsonpage**

### CODE REVIEW

**Comment by wilsonpage** [comment]
@arcturus r?

**Comment by wilsonpage** [comment]
I've implemented as much of this on the JS side as possible. I found displaying alerts/prompts outside of ReactNative framework was messing with timings. I think we should have the rule that all UI related work should be controlled by the JS layer.

File: android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewManager.java [hunk 1]
```diff
@@ -48,8 +48,6 @@ protected WebView createViewInstance(ThemedReactContext reactContext) {
         MagnetWebView magnetWebView = new MagnetWebView(mContext);
         reactContext.addLifecycleEventListener(magnetWebView);
 
-        Context otherContext = magnetWebView.getContext();
```
Comment (arcturus) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewManager.java:51]: What is this change? Is it related to this PR?

Comment (wilsonpage) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewManager.java:51]: Dead/unused variable I found.

**Comment by arcturus** [comment]
Works perfect in android n too, shiiiip it!

r+!

File: android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewManager.java (hunk 1 above)
Comment (arcturus) [inline android/app/src/main/java/org/mozilla/magnet/webview/MagnetWebViewManager.java:51]: Fair enough!

**Pull request merged**